src/traces/histogram2dcontour/attributes.js

```diff
@@ -38,7 +39,9 @@ module.exports = extendFlat({
         smoothing: contourAttrs.line.smoothing,
         editType: 'plot'
     },
-    zhoverformat: histogram2dAttrs.zhoverformat,
+    xhoverformat: axisHoverFormat('x'),
+    yhoverformat: axisHoverFormat('y'),
+    zhoverformat: axisHoverFormat('z'),
```

Comment:
```suggestion
    zhoverformat: axisHoverFormat('z', 1),
```
In fact contour, histogram2d, and histogram2dcontour could all take these from heatmap, right? Might be marginally simpler.